packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -150,7 +156,7 @@ const DMT5TradeModal = ({
             <div className='cfd-trade-modal__login-specs'>
                 <div className='cfd-trade-modal__login-specs-item'>
                     <Text className='cfd-trade-modal--paragraph'>{localize('Broker')}</Text>
-                    <SpecBox is_bold value={'Deriv Limited'} />
+                    <SpecBox is_bold is_broker value={'Deriv Holdings (Guernsey) Limited'} />
```

Comment:
```suggestion
                    <SpecBox is_bold is_broker value={localize('Deriv Holdings (Guernsey) Limited')} />
```

Reply:
Its a name and it should not be localized